spring-mvc-java/src/main/java/com/baeldung/spring/web/config/MyHybridAppInitializer.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.spring.web.config;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRegistration;
+
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.context.support.XmlWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+
+// copied from - spring-mvc-web-vs-initializer 
+public class MyHybridAppInitializer implements WebApplicationInitializer {
```

Comment:
@pivovarit  as I have mentioned in earlier comment, I have created this class here in spring-mvc-java to match the case of the article: http://www.baeldung.com/spring-xml-vs-java-config .